### PR TITLE
Fixed the url 404 not found for tutorial.md and changed the name

### PR DESCRIPTION
**Pull request by pedrobragadev**
#2137 

# Description

### Problem:
-  When you click on this url [tutorial](https://github.com/exercism/java/blob/main/exercises/hello-world/TUTORIAL.md) you will receive a 404 not found

### Solution:
- Get the real path for the tutorial and substitute all the old ones for the real url tutorial.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/shared/.docs/tests.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ If you get stuck, at any point, don't forget to reach out for [help](https://git
     ```
    *(Don't worry about the tests failing, at first, this is how you begin each exercise.)*
 
-4. Solve the exercise.  Find and work through the `TUTORIAL.md` guide ([view on GitHub](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/TUTORIAL.md)).
+4. Solve the exercise.  Find and work through the `instructions.append.md` guide ([view on GitHub](https://github.com/exercism/java/blob/master/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)).
```
Comment (andrerfcsantos) [inline exercises/shared/.docs/tests.md:111]:
```suggestion
4. Solve the exercise.  Find and work through the `instructions.append.md` guide ([view on GitHub](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)).
```

File: exercises/shared/.docs/tests.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ If you get stuck, at any point, don't forget to reach out for [help](https://git
     ```
    *(Don't worry about the tests failing, at first, this is how you begin each exercise.)*
 
-4. Solve the exercise.  Find and work through the `TUTORIAL.md` guide ([view on GitHub](https://github.com/exercism/java/blob/master/exercises/practice/hello-world/TUTORIAL.md)).
+4. Solve the exercise.  Find and work through the `instructions.append.md` guide ([view on GitHub](https://github.com/exercism/java/blob/master/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)).
```
Comment (andrerfcsantos) [inline exercises/shared/.docs/tests.md:76]:
```suggestion
4. Solve the exercise.  Find and work through the `instructions.append.md` guide ([view on GitHub](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)).
```

File: exercises/shared/.docs/tests.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ Choose your operating system:
      ```
    *(Don't worry about the tests failing, at first, this is how you begin each exercise.)*
 
-5. Solve the exercise.  Find and work through the `TUTORIAL.md` guide ([view on GitHub](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/TUTORIAL.md)).
+5. Solve the exercise.  Find and work through the `instructions.append.md` guide ([view on GitHub](https://github.com/exercism/java/blob/master/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)).
```
Comment (andrerfcsantos) [inline exercises/shared/.docs/tests.md:41]:
```suggestion
5. Solve the exercise.  Find and work through the `instructions.append.md` guide ([view on GitHub](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)).
```

File: exercises/practice/two-fer/.docs/instructions.append.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Instructions append
 
 Before you start, make sure you understand how to write code that can pass the test cases.
-For more context, check out this [tutorial](https://github.com/exercism/java/blob/master/exercises/hello-world/TUTORIAL.md).
+For more context, check out this [tutorial](https://github.com/exercism/java/blob/master/exercises/practice/hello-world/.docs/instructions.append.md#tutorial).
```
Comment (andrerfcsantos) [inline exercises/practice/two-fer/.docs/instructions.append.md:4]:
```suggestion
For more context, check out this [tutorial](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/.docs/instructions.append.md#tutorial).
```

File: exercises/practice/rna-transcription/.docs/instructions.append.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
 # Instructions append
 
 For more help on how to solve this exercise, please refer to the tutorial provided as part of the hello world exercise:
-[TUTORIAL.md](https://github.com/exercism/java/blob/master/exercises/hello-world/TUTORIAL.md)
+[instructions.append.md](https://github.com/exercism/java/blob/master/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)
```
Comment (andrerfcsantos) [inline exercises/practice/rna-transcription/.docs/instructions.append.md:4]:
```suggestion
[instructions.append.md](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)
```

File: exercises/practice/leap/.docs/instructions.append.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Instructions append
 
 Before you start, make sure you understand how to write code that can pass the test cases.
-For more context, check out this [tutorial](https://github.com/exercism/java/blob/master/exercises/hello-world/TUTORIAL.md).
+For more context, check out this [tutorial](https://github.com/exercism/java/blob/master/exercises/practice/hello-world/.docs/instructions.append.md#tutorial).
```
Comment (andrerfcsantos) [inline exercises/practice/leap/.docs/instructions.append.md:4]:
```suggestion
For more context, check out this [tutorial](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/.docs/instructions.append.md#tutorial).
```

File: exercises/practice/armstrong-numbers/.docs/instructions.append.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
 # Instructions append
 
 For more help on how to solve this exercise, please refer to the tutorial provided as part of the hello world exercise:
-[TUTORIAL.md](https://github.com/exercism/java/blob/master/exercises/hello-world/TUTORIAL.md)
+[instructions.append.md](https://github.com/exercism/java/blob/master/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)
```
Comment (andrerfcsantos) [inline exercises/practice/armstrong-numbers/.docs/instructions.append.md:4]:
```suggestion
[instructions.append.md](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/.docs/instructions.append.md#tutorial)
```

**Review by andrerfcsantos — CHANGES_REQUESTED** [review]
The main branch is now called `main` and not `master`. Github does redirect `master` to `main` automatically, but let's not rely on that behavior.

**Comment by pedrobragadev** [comment]
You are right. I already changed. Can I create another PR so I can change all the "master" path to "main"?

**Comment by andrerfcsantos** [comment]
@pedrobragadev Go ahead! If possible, while doing that, would get great to check if the links are still valid. I imagine some of them might refer files that don't exist.

**Pull request merged**